geoips/schema/product_defaults/bases/colormapper.yaml

```diff
@@ -16,7 +20,7 @@ properties:
       arguments:
         type: object
         unevaluatedProperties: false
-        properties:  # These are all optional arguments, matches "matplotlib" family
+        properties: # These are all optional arguments, matches "matplotlib" family
```

Comment:
Need to add descriptions for every field, so schema are self documenting.

Reply:
Do you mean that we need more comments like this?

Reply:
We can do this in another PR or series of PRs, but the idea would be to add `description:` to every property.  So, `data_range:` might look like this:
```
          data_range:
            description: "The minimum and maximum data values to use in normalization or clipping."
            type: array
            minItems: 2
            maxItems: 2
            items:
              type: number